cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PrioritizeOneAboveMinIsrWithOfflineReplicasStrategy.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.executor.strategy;
+
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionTask;
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionUtils;
+import org.apache.kafka.common.Cluster;
+import org.apache.kafka.common.PartitionInfo;
+import java.util.Comparator;
+import java.util.Set;
+import java.util.TreeSet;
+
+
+/**
+ * The strategy, which tries to move replicas of OneAboveMinISR partitions with offline replicas.
+ * When multiple brokers are offline, OneAboveMinISR partitions are at a higher risk than partitions with no offline replicas.
+ * This strategy aims to help fixing such partitions faster.
+ * When using this strategy, always chain it after {@link PrioritizeMinIsrWithOfflineReplicasStrategy},
+ * which aims to help fixing higher risk (At/Under)MinISR.
```

Comment:
Nit: `always chain` -> `it is recommended to chain`
Because if, for some reason, users want to use this strategy w/o chaining it, CC still supports it.